packages/sdk/src/restclient.spec.ts

```diff
@@ -169,15 +170,61 @@ describe("RestClient", () => {
     expect(client).toBeTruthy();
   });
 
-  describe("nodeInfo", () => {
-    it("works", async () => {
+  // The /auth endpoints
+
+  describe("authAccounts", () => {
+    it("works for unused account without pubkey", async () => {
       pendingWithoutWasmd();
       const client = new RestClient(httpUrl);
-      const info = await client.nodeInfo();
-      expect(info.node_info.network).toEqual(defaultNetworkId);
+      const { result } = await client.authAccounts(unusedAccount.address);
```

Comment:
This part is just unrelated test cleanup, right?
Seems solid, but I don't see it to do with min/maxHeight (or am I missing something?)

Reply:
Oh, this is due to a bad diff algorithm... the code was here before

Reply:
Yeah a lot of diff was created by alphabetically sorting the endpoints